GVRf/Framework/src/org/gearvrf/scene_objects/GVRConeSceneObject.java

```diff
@@ -23,8 +23,8 @@
 public class GVRConeSceneObject extends GVRSceneObject {
 
     private static final String TAG = "GVRConeSceneObject";
-    private static final int NUM_STACKS = 2;
-    private static final int NUM_SLICES = 36;
+    private static final int STACK_NUMBER = 10;
```

Comment:
This is OK (and better than `NUM`) but I do think `STACK_COUNT` is better (not just shorter) than `STACK_NUMBER`. I am _not_ asking you to change it, though!